docs.phtml

```diff
@@ -180,15 +180,15 @@
     </div>
   </div> <!-- stabledoc -->
 
-  <p class="justify"><?php echo T_("This is the latest stable release of the <span class=\"gnucash\">GnuCash</span> documentation. <em>Stable</em> means that this documentation is not necessarily the most up to date but it works without any problem.");?></p>
+  <p class="justify"><?php echo T_("This is the latest stable release of the <span class=\"gnucash\">GnuCash</span> documentation, meaning not necessarily up-to-date, but works without problems.");?></p>
```

Comment:
I prefer the original here.

Reply:
I prefer not explaining this in an apologetic manner and rather just keep the documentation up-to-date. Just shorter is the best overall benefit to be had for now is what I think.

Reply:
How is your version "less apologetic"?!?!?!? If you're balking at the explanation of what "stable documentation" means, then remove the sentence altogether, and leave this as "This is the latest stable release of the GnuCash documentation."

However, I sincerely believe (after 15 years of using gnucash) that most regular users are unfamiliar with what "stable" means generally, and are unlikely to understand this project's particular interpretation of the term.

Reply:
It isn't. Good idea. Agreed.